js/analyses/calculateFleschReading.js

```diff
@@ -0,0 +1,32 @@
+/** @module analyses/calculateFleschReading */
+
+var cleanText = require( "../stringProcessing/cleanText.js" );
+var stripNumbers = require( "../stringProcessing/stripNumbers.js" );
+var stripHTMLTags = require( "../stringProcessing/stripHTMLTags.js" );
+var countSentences = require( "../stringProcessing/countSentences.js" );
+var countWords = require( "../stringProcessing/countWords.js" );
+var countSyllables = require( "../stringProcessing/countSyllables.js" );
+
+/**
+ * This calculates the fleschreadingscore for a given text
+ * The formula used:
+ * 206.835 - 1.015 (total words / total sentences) - 84.6 ( total syllables / total words);
+ *
+ * @param {string} text The text to analyze the fleschreading score.
+ * @returns {number} the score of the fleschreading test
+ */
+module.exports = function( text ) {
+	if ( typeof text === "undefined" || text === "" ) {
```

Comment:
We decided to remote the checks for "undefined" from the analyses so this one should be removed as well.